File: app/src/main/java/com/satugenggam/aplikasi1/data/model/LoginResponse.java
package com.satugenggam.aplikasi1.data.model;

import com.google.gson.annotations.SerializedName;

public class LoginResponse{

	@SerializedName("access_token")
	private String accessToken;

	@SerializedName("token_type")
	private String tokenType;

	@SerializedName("expires_in")
	private int expiresIn;

	@SerializedName("user")
	private UserLoginResponse userLoginResponse;

	public String getAccessToken(){
		return accessToken;
	}

	public String getTokenType(){
		return tokenType;
	}

	public int getExpiresIn(){
		return expiresIn;
	}

	public UserLoginResponse getUserLoginResponse(){
		return userLoginResponse;
	}
}

File: app/src/main/java/com/satugenggam/aplikasi1/data/model/LoginResponse.kt
package com.satugenggam.aplikasi1.data.model


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class LoginResponse(

	@field:SerializedName("access_token")
	var accessToken: String? = null,

	@field:SerializedName("token_type")
	var tokenType: String? = null,

	@field:SerializedName("expires_in")
	var expiresIn: Int? = null,

	@field:SerializedName("user")
	val userLoginResponse: UserLoginResponse? = null
) : Parcelable
